domains/abhishek.is-not-a.dev.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "../schemas/domain.schema.json",
+
+  "description": "Project Description",
+
+  "domain": "is-not-a.dev",
+  "subdomain": "abhishek",
+
+  "owner": {
+    "email": "ramabishek40@gmail.com"
+  },
+
+  "record": {
+    "NS": ["chip.ns.cloudflare.com", "jacqueline.ns.cloudflare.com"]
```

Comment:
Sorry, we don't unfortunately support Cloudflare for nameservers yet.

Reply:
I have removed the NS field and added and A record field instead. Please do check and tell me if it's valid.